sheetstorm/management/commands/generate_test_data.py

```diff
@@ -65,7 +93,7 @@ def _get_user_options(options: OptionsDictType) -> Dict[str, Optional[int]]:
 
     @staticmethod
     def _get_superuser_request(options: Dict[str, Any]) -> bool:
-        is_superuser_create_request = options[superuser_user_type]
+        is_superuser_create_request = options[SUPERUSER_USER_TYPE]
```

Comment:
Changes like here i.e. changing variables are not related with this commit and should be in different one. Please extract changes for variable name refactoring to separate commit. This will require to amend commit and extract these changes from this commit.